backend/runtimes/runtimes.go

```diff
@@ -1,18 +1,85 @@
 package runtimes
 
 import (
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/internal/backend/httpsvc"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	configruntimesvc "go.autokitteh.dev/autokitteh/runtimes/configrt/runtimesvc"
 	pythonruntime "go.autokitteh.dev/autokitteh/runtimes/pythonrt"
 	starlarkruntimesvc "go.autokitteh.dev/autokitteh/runtimes/starlarkrt/runtimesvc"
 	"go.autokitteh.dev/autokitteh/sdk/sdkruntimes"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.uber.org/zap"
 )
 
-func New() sdkservices.Runtimes {
-	return kittehs.Must1(sdkruntimes.New([]*sdkruntimes.Runtime{
+type Config struct {
+	RemoteRunnerEndpoints []string `koanf:"remote_runner_endpoints"`
+	RemoteRunner          bool     `koanf:"enable_remote_runner"`
+	WorkerAddress         string   `koanf:"worker_address"`
+	// TODO: This is a hack to prevent running configure on pythonrt in each test
+	// which currently install venv everytime and takes a really long time
+	// need to find a way to share the venv once for all tests
+	PythonRTEnabled bool `koanf:"enable_python_runtime"`
+}
+
+var Configs = configset.Set[Config]{
+	Default: &Config{
+		RemoteRunnerEndpoints: []string{"localhost:9291"},
+		RemoteRunner:          false,
+		PythonRTEnabled:       true,
+	},
+	Test: &Config{
+		PythonRTEnabled: false,
+	},
+}
+
+func New(cfg *Config, l *zap.Logger) sdkservices.Runtimes {
+	runtimes := []*sdkruntimes.Runtime{
 		starlarkruntimesvc.Runtime,
 		configruntimesvc.Runtime,
-		pythonruntime.Runtime,
-	}))
+	}
+
+	//TODO: need to rethink
+	if cfg == nil {
+		cfg = Configs.Default
+	}
+
+	if cfg.PythonRTEnabled {
+		runtimes = append(runtimes, pythonruntime.Runtime)
+		pythonruntime.ConfigureWorkerGRPCHandler(l)
+	}
+
+	return kittehs.Must1(sdkruntimes.New(runtimes))
+}
+
+func Configure(cfg *Config, l *zap.Logger, svc httpsvc.Svc) {
+	if !cfg.PythonRTEnabled {
+		return
+	}
+	if cfg.WorkerAddress == "" {
+		cfg.WorkerAddress = svc.Addr()
+	}
+
+	if cfg.RemoteRunner {
+		if len(cfg.RemoteRunnerEndpoints) == 0 {
+			panic("remote runner is enabled but no runner endpoints provided")
+		}
+		if err := pythonruntime.ConfigureRemoteRunnerManager(pythonruntime.RemoteRuntimeConfig{
```

Comment:
I prefer:

```go
err = pythonruntime.ConfigureRemoteRunnerManager( .... )
if err != nil {
...
```

Reply:
I tend to not change stuff on personal preferences unless they are convention or has real world implications, 
is this one of them ?

Reply:
It's not a personal preference, it's good style. Multi-line `if` statements are confusing and hard to understand. The shortcut of joining `x := y` and `if x` into `if x := y; x` is useful only if it fits comfortably in a single line. If not, it's better to keep them separate, especially if `y` is multi-line on its own.